htdocs/reception/card.php

```diff
@@ -1164,6 +1184,9 @@
 					$line->fetch_optionals();
 
 					$line->array_options = array_merge($line->array_options, $srcLine->array_options);
+					if (empty($line->array_options) && !empty($dispatchLines[$indiceAsked]['array_options'])) {
+						$line->array_options = $dispatchLines[$indiceAsked]['array_options'];
```

Comment:
So if there is some extrafields submited into form, it will erase completely all $line->array_options ?
May be we can make this 3 new lines before the line 1186 instead of after ?
So submited data are added,  and extrafields coming from $srcLine are also used (current behavior).

Reply:
You’re right, I moved the lines according to your suggestion.